src/components/KMenu/KMenu.vue

```diff
@@ -92,32 +92,32 @@ const proceed = (): void => {
 @import '@/styles/functions';
 
 .k-menu {
-  background-color: var(--white);
-  border: 1px solid var(--grey-300);
-  border-radius: 4px;
-  padding-bottom: 8px;
-  padding-top: 11px;
+  background-color: var(--white, var(--kui-color-background, $kui-color-background));
+  border: var(--kui-border-width-10, $kui-border-width-10) solid var(--grey-300, var(--kui-color-background-neutral-weak, $kui-color-background-neutral-weak));
+  border-radius: var(--KCardBorderRadius, var(--kui-border-radius-20, $kui-border-radius-20));
+  padding-bottom: var(--kui-space-40, $kui-space-40);
+  padding-top: var(--kui-space-50, $kui-space-50);
 }
 
 .clear-cta-button > :deep(button.k-button) {
   border: none;
-  color: var(--blue-300);
-  font-size: 13px;
-  font-weight: 500;
-  line-height: 13px;
-  margin-bottom: 6px;
-  margin-top: 10px;
-  padding-top: 2px;
+  color: var(--blue-300, var(--kui-color-border-primary-weak, $kui-color-border-primary-weak));
```

Comment:
issue: `color-border` in `color` property

Reply:
updated to use `kui-color-background-primary-weak` instead since don't have `kui-color-text-primary-weak` or anything close to `blue.30`

Reply:
added and update to use `$tmp` token